app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/address/PropertyDetailsController.scala

```diff
@@ -590,7 +635,17 @@ class PropertyDetailsController @Inject() (
             .fold(
               formWithErrors =>
                 BadRequest(
-                  multipleDisposalsAcquisitionPricePage(formWithErrors, backLink, r.journey.subscribedDetails.isATrust)
+                  multipleDisposalsAcquisitionPricePage(
+                    formWithErrors,
+                    backLink,
+                    r.journey.subscribedDetails.isATrust,
+                    r.individualUserType match {
```

Comment:
might be worth sticking this as a separate method seeing as you're doing the same thing multiple times